File: test/new_lang/test5/TestProgram.pas
program TestProgram;
uses SplashKit;

procedure Main();
var
  arr: array of Boolean;
begin
  arr := GetBools();
  PrintAll(arr);
  WriteLn('Adding false');
  AddBool(arr, false);
  PrintAll(arr);
end;

begin
  Main();
end.

File: test/new_lang/test5/test_program.cpp
#include <vector>
#include <iostream>
using namespace std;

#include <splashkit.h>

int main()
{
  vector<bool> arr;

  arr = get_bools();
  print_all(arr);
  cout << "Adding false" << endl;
  add_bool(arr, false);
  print_all(arr);
}
